contrib/vmap_extractor/vmapextract/adtfile.cpp

```diff
@@ -168,12 +167,16 @@ bool ADTFile::init(uint32 map_num, uint32 tileX, uint32 tileY, StringSet& failed
                 char* p = buf;
                 while (p < buf + size)
                 {
-                    fixnamen(p, strlen(p));
-                    string path(p);                         // Store copy after name fixed
+                    std::string path(p);
+
+                    char* s = GetPlainName(p);
+                    FixNameCase(s, strlen(s));
```

Comment:
Looks like duplicate code. Name is normalized here and also inside ExtractSingleModel(..)

thats why ModelInstanceNames.emplace_back(fixedName); was after  ExtractSingleModel(path, fixedName, failedPaths);